forward_engineering/ddlProvider/templates.js

```diff
@@ -134,4 +134,7 @@ module.exports = {
 	setSequenceType: 'ALTER SEQUENCE IF EXISTS ${name} SET ${sequenceType};\n',
 
 	alterSequence: 'ALTER SEQUENCE IF EXISTS ${name}' + '${options}' + ';\n',
+
+	updateColumnDefaultValue: 'ALTER TABLE ${tableName} ALTER COLUMN ${columnName} SET DEFAULT ${defaultValue};',
```

Comment:
https://github.com/hackolade/PostgreSQL/blob/21ba14993188300d00bd5b308d9feb9122c60114/forward_engineering/ddlProvider/templates.js#L32-L42

The rest of the alter table templates use the `IF EXISTS` clause, perhaps it’s worth adding it here too